CODING_STYLE.md

```diff
@@ -0,0 +1,39 @@
+# Coding Style
+
+## General
+
+When contributing to the project, and if otherwise not mentioned in this document, our coding conventions
+follow the Microsoft [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions)
+and standard [Naming Guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines).
+
+## Class Members
+
+Members and types should always have the lowest possible visibility.
+
+Ordering of class members should be the following:
+1. Constants
+2. Fields
+3. Abstract members
+4. Constructors
+5. Methods and Properties
+6. Private nested classes
+
+Furthermore, each of these categories should be ordered from higher to lower accessibility level (public, internal, protected, private).
+
+## FIXME and ToDo
+
+`FIXME` should only be used in the code during development as a temporary reminder that there is still work to
```

Comment:
I'd like to be explicit about the double colon, because `FIXME This is not finished` does work as well and it's not nice/consistent. I would keep the title without it, but the text full of it.
```suggestion
`FIXME:` should only be used in the code during development as a temporary reminder that there is still work to
```

Reply:
I'd rather not to be that strict here. I personally never put colons in FIXME comments and I think being that strict for code that is under work and should theoretically not be reviewed and merged to master is a bit overkill